tests/integration_tests/build/test_binary_static_linking.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests that check if the release binary are statically linked.
+
+"""
+
+import pytest
+import host_tools.cargo_build as host
+from framework import utils
+
+@pytest.mark.timeout(500)
+def test_firecracker_binary_static_linking():
+    """
+    Test to make sure the firecracker binary is statically linked.
+    """
+    fc_binary_path = host.get_binary("firecracker")
+    result = utils.run_cmd(f"file {fc_binary_path}")
+    assert "statically linked" not in result
```

Comment:
The release binary is not build by `hot.get_binary()`, we probably also want to put this assertion at the end of `release.sh` (btw, shouldnt it be `"statically linked in results"`, instead of `not in`?)

Reply:
as per https://github.com/firecracker-microvm/firecracker/blob/main/tests/host_tools/cargo_build.py#L79 it looks like host.get_binary() is returning the release binary, am I missing something?
and yeah the previous check were not correct, I have updated it thanks for pointing out.

Reply:
> as per https://github.com/firecracker-microvm/firecracker/blob/main/tests/host_tools/cargo_build.py#L79 it looks like host.get_binary() is returning the release binary, am I missing something?

host.get_binary() is producing a release profile binary, but our _release script_ does not use host.get_binary() to get its release-profile binary :D But generally, "binary compiled with --release flag" and "binary is statically linked" are independent of each other, so even if we get a statically linked binary from host.get_binary() does not tell us anything about the binary that release.sh is producing
Sorry, should've been clearer.

Reply:
update the commit msg to reflect this.